jsonpath.go

```diff
@@ -550,6 +555,40 @@ func get_filtered(obj, root interface{}, filter string) ([]interface{}, error) {
 	return res, nil
 }
 
+func get_scan(obj interface{}) (interface{}, error) {
+	if reflect.TypeOf(obj) == nil {
+		return nil, ErrGetFromNullObj
+	}
+	switch reflect.TypeOf(obj).Kind() {
+	case reflect.Map:
+		var res []interface{}
+		if jsonMap, ok := obj.(map[string]interface{}); ok {
+			for _, v := range jsonMap {
+				res = append(res, v)
+			}
+			return res, nil
+		}
+		for _, kv := range reflect.ValueOf(obj).MapKeys() {
+			res = append(res, reflect.ValueOf(obj).MapIndex(kv).Interface())
+		}
```

Comment:
Slightly better as:

```suggestion
		iter := reflect.ValueOf(obj).MapRange()
		for iter.Next() {
			res = append(res, iter.Value().Interface())
		}
```

?